_commands/basics/uptime.md

```diff
@@ -0,0 +1,37 @@
+---
+---
+
+uptime
+------
+
+This command shows the current time followed by your computer's uptime.
+
+~~~ bash
+$ uptime
+3:03 up 1 day, 6 hrs, 2 users, load averages: 1.38 1.43 1.42
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### Load Times
```

Comment:
Move this and the explanation below it to the bottom of the page. I'm not sure "Load Times" is the best title for this also